services/uploads/src/lib/otel.ts

```diff
@@ -43,6 +43,7 @@ export function initMeter(serviceName: string) {
         })
     )
     const metricReader = new PeriodicExportingMetricReader({
+        // TODO: Changes need to be made here
```

Comment:
@mojotalantikite Do you have insight on how we might want to resolve this? 

We could use something like ConsoleMetricExporter that has the appropriate type, but I think that changes the functionality. I thought this might be a versioning issue, but didn't explore that very deeply.